source/math/FloatComplexVectorList.ooc

```diff
@@ -134,18 +133,20 @@ FloatComplexVectorList: class extends VectorList<FloatComplex> {
 				result[i] = evenOutput[i] + root * oddOutput[i]
 				result[halfLength + i] = evenOutput[i] - root * oddOutput[i]
 			}
+			evenInput free()
+			oddInput free()
```

Comment:
I see a potential for doing this work in-place, which would greatly reduce memory allocation and deallocation... probably shouldn't spend time on that right now though, but it would be something to consider in the future.